tests/puppeteer/campaigns/sanity/products/01_Filter_products.js

```diff
@@ -0,0 +1,78 @@
+// Using chai
+const {expect} = require('chai');
+
+// importing pages
+const BO_LOGIN_PAGE = require('../../../pages/BO/login');
+const BO_DASHBOARD_PAGE = require('../../../pages/BO/dashboard');
+const BOBasePage = require('../../../pages/BO/BObasePage');
+const ProductPage = require('../../../pages/BO/product');
+
+let page;
+let BO_LOGIN;
+let BO_DASHBOARD;
+let boBasePage;
+let productPage;
+let numberOfProducts;
+
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  BO_LOGIN = await (new BO_LOGIN_PAGE(page));
```

Comment:
Same here, is it normal to not have the same caps ?

Reply:
done thanks